src/main/java/hudson/plugins/jira/versionparameter/JiraVersionParameterDefinition.java

```diff
@@ -159,5 +181,12 @@ public Result(final Version version) {
             this.name = version.getName();
             this.id = version.getId();
         }
+
+        @Override
+        public boolean equals(Object o) {
```

Comment:
API Note:

It is generally necessary to override the [hashCode](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html) method whenever this method is overridden, so as to maintain the general contract for the hashCode method, which states that equal objects must have equal hash codes.